src/server/game/Chat/Chat.h

```diff
@@ -111,6 +112,7 @@ class TC_GAME_API ChatHandler
         virtual LocaleConstant GetSessionDbcLocale() const;
         virtual int GetSessionDbLocaleIndex() const;
 
+        bool IsHumanReadable() const { return humanReadable; }
```

Comment:
Right, this was supposed to be virtual and returning true and `bool humanReadable` moved to AddonCliHandler